test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- According to the project's requirement, you are to ignore your test.md file, just like you ignored the node_modules. Kindly add test.md to the .gitignore file 🍒

Reply:
@Favourezeugwa I have fixed the error according to your direction.